Added Teaser Video For Home Page And Name to Contributors

speedytechdev opened the pull request:
[Happy_Hacking-master.zip](https://github.com/sudz123/Happy_Hacking/files/2453026/Happy_Hacking-master.zip)

sudz123 commented:
I am not sure what you have done, but if you had done the right thing then here it should only show 2-3 files changed and 2-3 of your commits. Kindly take a look at how to submit a PR and redo this. I am not closing this one till you make a patch.

sudz123 committed:
Merge pull request #450 from HemangiS/master

added home page links to html pages

sudz123 committed:
Merge branch 'master' into master

sudz123 committed:
Merge pull request #457 from abhi010101/master

added bootstrap html boilerplate

sudz123 committed:
Merge branch 'master' of https://github.com/yawningpaws/Happy_Hacking into yawningpaws-master

sudz123 committed:
Merge branch 'yawningpaws-master'

sudz123 committed:
Merge branch 'master' into master

sudz123 committed:
Merge pull request #460 from gigishankarunarathne/master

updated readme file and added my name to contributors file

sudz123 committed:
Merge branch 'pr/Halloweeninfo' of https://github.com/Malvikabhalla99/Happy_Hacking into Malvikabhalla99-pr/Halloweeninfo

sudz123 committed:
Merge branch 'Malvikabhalla99-pr/Halloweeninfo'

gormleymark committed:
Update README.md

added section pointing users to the more advanced Optimizer project

sudz123 committed:
Merge pull request #465 from gormleymark/master

Update README.md

gormleymark committed:
Update Contributors.md

added my name

gormleymark committed:
Update home.html

added my name with link to my profile

ManiacalBilby committed:
Added trick or treat link to menubar

Commit:
Removed duplicated section ids in halloween.html

felicia-goh committed:
Edited grammar, links, <script> position

- Edited grammar and break positions of the "How to request your shirt?" section
- Shifted .js scripts to the bottom of the <body> tag
- Added 'target = "_blank"' for all links to contributors' pages

ArpitaGanatra committed:
Added buttons in sudz.html

Commit:
Added drop-shadow on images.

Commit:
Added personal page

Commit:
changed line

ankitsuman4 committed:
Added Home Link in some HTML

Harindu973 committed:
I added my name

cassmtnr committed:
added cassianomon

deshanjali committed:
updated margin and the padding

haiderkhaleel23 committed:
Added Contribution

michaelmatherson committed:
Update Contributors.md

added my name to contributors list

UmmeAmmaraa committed:
Add Umme Ammara to Contributors list

Commit:
Merge pull request #479 from UmmeAmmaraa/add-ammara

Add Umme Ammara to Contributors list

Commit:
Merge branch 'master' into patch-1

Commit:
Merge pull request #478 from michaelmatherson/patch-1

Update Contributors.md

Commit:
Merge branch 'master' into contribute

Commit:
Merge pull request #477 from haiderkhaleel23/contribute

Added Contribution

Commit:
Merge branch 'master' into master

Commit:
Merge pull request #466 from gormleymark/master

added my name

Commit:
Merge pull request #467 from ManiacalBilby/add-trickortreat-to-menubar

Added trick or treat link to menubar

sudz123 committed:
Merge branch 'master' into master

sudz123 committed:
Merge pull request #474 from Harindu973/master

I added my name

sudz123 committed:
Merge pull request #473 from ankitsuman4/master

Added Home Link in some HTML

sudz123 committed:
Merge pull request #471 from ArpitaGanatra/master

Added buttons in sudz.html

sudz123 committed:
Merge pull request #469 from yawningpaws/patch-1

Edited grammar, links, <script> position

sudz123 committed:
Merge pull request #470 from KostasLib/add-drop-shadow

Added drop-shadow on images.

sudz123 committed:
Merge pull request #475 from cassianomon/master

added cassianomon

sudz123 committed:
Merge pull request #472 from heggrik/master

Added personal page

sudz123 committed:
Merge pull request #476 from deshanjali/patch-1

updated margin and the padding

sudz123 committed:
Merge pull request #468 from szabolcsnagy/fix_section_ids

Removed duplicated section ids in halloween.html

Commit:
add my name

Commit:
add my name

Commit:
Fix: Trick or treat menu item now linked to the corresponding section

ManiacalBilby committed:
Added name and info to Contributors.md

Commit:
Merge pull request #483 from szabolcsnagy/fix_trickortreat_link

Fix: Trick or treat menu item now linked to the corresponding section

Commit:
Merge pull request #484 from ManiacalBilby/add-name-to-contributors

Added name and info to Contributors.md

rahma-ba committed:
Fix docs

howieandersen committed:
Added personal page, and name to contributors

surayyayousaf committed:
add me to contributors list

Commit:
home page

darkdog555 committed:
Update README.md

Ananthu-Gopi committed:
Updated sudz.html

Added  input validation for email in the text field

sudz123 committed:
Merge pull request #500 from Ananthu-Gopi/patch-2

Updated sudz.html

sudz123 committed:
Merge pull request #490 from howieandersen/master

Added personal page, and name to contributors

sudz123 committed:
Merge pull request #497 from ravit2839/master

home page

sudz123 committed:
Merge pull request #492 from surayyayousaf/patch-1

add me to contributors list

sudz123 committed:
Merge pull request #491 from yogaap24/yogaap24

Yogaap24

sudz123 committed:
Merge pull request #488 from rahma-ba/patch-1

Fix docs

sudz123 committed:
Merge pull request #499 from darkdog555/master

Update README.md

Ananthu-Gopi committed:
Update sudz.html

1.added an image of  Sudarshan Kamath
2.modified the form elements and reduced the size occupied by the form elements
3.made buttons as round-edged  and added a little space between them
Please review my code .

sudz123 committed:
Merge pull request #504 from Ananthu-Gopi/patch-3

Update sudz.html

AlithAnar committed:
Fix broken zoom effect on images

AlithAnar committed:
Update contributors

bluejay2011 committed:
add SJ as contributor (#507)

* add SJ as contributor

* Update Contributors.md

Commit:
Merge branch 'master' into enhance/update-contributors

Commit:
Merge pull request #506 from AlithAnar/enhance/update-contributors

Update contributors

Rajani169 committed:
Update Contributors.md

Added my profile link

mon555 committed:
Update Contributors.md

flechnical committed:
Add new contributor

nitrocoder234 committed:
Added my name to the contributors list

iamsid2 committed:
Portfolio Page Added

amulyagaur committed:
Update Contributors.md

Added myself as contributor.

Commit:
added Lenny to Contributer List

Commit:
added zoom-effect on hover for images

pankajgoyal1 committed:
Added myself In Contributors

Commit:
Java start guide spanish version

JessicaMP committed:
add my version html

farzanfa00 committed:
add me

Sagar2366 committed:
Update Contributors.md

vinnyA3 committed:
feat(home.html): format card component markup

Formatted the card component markup.  In addition, moved inline-styles,
and changed some classes to improve card maintainability.

vinnyA3 committed:
feat(home.css): card component styling

Displaying the cards as blocks now, and added explicit height to
cards (since there are only two).  In addition, added a little
cosmetic styling (but did not alter the card's *original* structure).

I also added a media query to help with card responsiveness

vinnyA3 committed:
docs(contrib): add name :)

ronishz committed:
Update Contributors.md

prateek552 committed:
added my name

Commit:
Merge pull request #508 from Rajani169/patch-1

Update Contributors.md

Commit:
Merge pull request #511 from mon555/patch-1

Update Contributors.md

Commit:
Merge pull request #512 from flechnical/patch-1

Add new contributor

Commit:
Merge branch 'master' into master

Commit:
Merge pull request #513 from nitrocoder234/master

Added my name to the contributors list

Commit:
Merge pull request #514 from iamsid2/master

iamsid2 Portfolio Page Added

Commit:
Merge branch 'master' into patch-1

Commit:
Merge pull request #515 from amulyagaur/patch-1

Update Contributors.md

sudz123 committed:
Merge pull request #505 from AlithAnar/fix/restore-img-zoom-effect

Fix broken zoom effect on images

sudz123 committed:
Merge branch 'master' into master

sudz123 committed:
Merge pull request #525 from prateek552/master

added my name

sudz123 committed:
Merge pull request #524 from ronishz/master

Update Contributors.md

sudz123 committed:
Update Contributors.md

sudz123 committed:
Merge branch 'master' into patch-1

sudz123 committed:
Merge pull request #522 from Sagar2366/patch-1

Update Contributors.md

sudz123 committed:
Merge branch 'master' into patch-1

sudz123 committed:
Merge pull request #521 from farzanfa00/patch-1

add me

sudz123 committed:
Merge pull request #519 from camiloibarrayepes/camiloibarrayepes

Java start guide spanish version

sudz123 committed:
Merge pull request #520 from JessicaMP/jessica

add my version html

sudz123 committed:
Merge branch 'master' into master

sudz123 committed:
Merge pull request #518 from pankajgoyal1/master

Added myself In Contributors

sudz123 committed:
Merge branch 'master' into master

sudz123 committed:
Merge branch 'vinnyA3-master'

sudz123 committed:
Merge branch 'master' into add-halloween

sudz123 committed:
Merge branch 'lenni108-add-halloween'

Quarantinex committed:
Update README.md

aa-deet-eeya committed:
Removed the the box around the witch

kaushikpraveen committed:
Fixed issue #526

Removed small box-shadow around the witch. Please merge this PR
This is the right one!

athuljayaram committed:
Update Contributors.md

Platyka committed:
first commit

Platyka committed:
add to contributors

gofornaman committed:
Update Contributors.md

TohaAssegaf committed:
Update LICENSE.txt

1995Pradeep committed:
Add <pradeep> to Contributors list

venushaamarathunga committed:
My happy_hacking code

chamithopatha committed:
my code is here

confixyu committed:
happy condig xD

KashyapAnkur committed:
Update todolist.js

Manjulika77 committed:
Added HTML doctype declaration

Added HTML doctype declaration

Nayan942 committed:
7th commit

vinnyA3 committed:
feat(home.html): add social link markup

Moved the social links into a component to help with maintainability
and possible reusability.

vinnyA3 committed:
feat(home.html): add social link markup

Moved the social links into a component to help with maintainability
and possible reusability.

vinnyA3 committed:
feat(home): space in card + social link styles

Reduced the spacing between card sub elements and added styles
for social link component (to help with maintainability).

In addition, I added some base styles for link elements.

vinnyA3 committed:
feat(home): space in card + social link styles

Reduced the spacing between card sub elements and added styles
for social link component (to help with maintainability).

In addition, I added some base styles for link elements.

vinnyA3 committed:
Merge branch 'master' of github.com:vinnyA3/Happy_Hacking

vinnyA3 committed:
Merge branch 'master' of github.com:vinnyA3/Happy_Hacking

vinnyA3 committed:
fix(home.css): link color inheritance fix

Commit:
Added documentation to JS section
- Closures summary with code example added

Commit:
Updated doc formatting

jameslongs37 committed:
Update Contributors.md

added my name and profile link

Commit:
Updated doc formatting

Commit:
Updated doc formatting

melvinabraham committed:
Updated Contributer list

mounika0210 committed:
Changed animation css

edwinfrancis002 committed:
Update Contributors.md

johanreji committed:
Update Contributor.md

nickhuangClimax committed:
Add Nick Huang to contributors list

Utkarsh1c committed:
Added new details

added new contributor details to the file.

rohangaonk committed:
Update README.md

sudz123 committed:
Merge pull request #528 from Quarantinex/patch-1

Update README.md

sudz123 committed:
Merge pull request #533 from warmachine0609/patch-1

Update Contributors.md

sudz123 committed:
Merge pull request #536 from venushaamarathunga/my-hack

My happy_hacking code

sudz123 committed:
Merge branch 'master' into master

sudz123 committed:
Merge pull request #550 from melvinabraham/master

Updated Contributer list

sudz123 committed:
Merge branch 'master' into master

sudz123 committed:
Merge pull request #552 from edwinfrancis002/master

Update Contributors.md

sudz123 committed:
Merge branch 'master' into patch-1

sudz123 committed:
Merge branch 'master' into patch-1

sudz123 committed:
Merge pull request #548 from jameslongs37/patch-1

Update Contributors.md

sudz123 committed:
Merge branch 'master' into patch-1

sudz123 committed:
Merge pull request #531 from juniorhero/patch-1

Update Contributors.md

Babarbwn123 committed:
Babar Shabbir

sudz123 committed:
Merge branch 'master' into patch-1

sudz123 committed:
Merge pull request #553 from johanreji/patch-1

Update Contributor.md

sudz123 committed:
Merge pull request #529 from aa-deet-eeya/removing-witch-box

Fixed issue #526 (Removed the the box around the witch)

sudz123 committed:
Merge pull request #530 from kaushikpraveen/patch-2

Fixed issue #526

adnanpak committed:
adnanrana

sudz123 committed:
Merge branch 'master' into platy

sudz123 committed:
Merge pull request #532 from Platyka/platy

first commit

sudz123 committed:
Merge pull request #535 from gariya95/pradeep

 hi Add <pradeep> to Contributors list

sudz123 committed:
Merge pull request #534 from TohaAssegaf/patch-1

Update LICENSE.txt

sudz123 committed:
Merge branch 'master' into kanda6th

sudz123 committed:
Merge pull request #537 from kanda94/kanda6th

my code is here

sudz123 committed:
Merge branch 'master' into happycoding_confix

sudz123 committed:
Merge pull request #538 from confixyu/happycoding_confix

happy condig xD

sudz123 committed:
Merge pull request #540 from KashyapAnkur/master

Update todolist.js

sudz123 committed:
Merge branch 'master' into master

sudz123 committed:
Merge pull request #541 from Manjulika77/master

Added HTML doctype declaration

sudz123 committed:
Merge branch 'master' into master

sudz123 committed:
Merge pull request #542 from Nayan942/master

7th commit

sudz123 committed:
Merge pull request #546 from vinnyA3/master

feat(home): reduce card content spacing + move social links into own component

Kaustubhprabhu98 committed:
Basic EMI calculator

Commit:
Merge branch 'master' into add-babar

Commit:
Merge pull request #560 from Babarbwn123/add-babar

Babar Shabbir

Commit:
Merge branch 'master' into add-adnanrana

Commit:
Merge pull request #559 from adnanpak/add-adnanrana

adnanrana

mshoaib85 committed:
add me to contributors list

vatsalvv committed:
makin-a-new-world

sudz123 committed:
Merge pull request #549 from jrcarney/closures-added-to-js-doc

Closures added to js doc

sudz123 committed:
Merge pull request #551 from mounika0210/master

Changed animation css

sudz123 committed:
Merge branch 'master' into feature/add_nick_huang

sudz123 committed:
Merge pull request #554 from NickHuangDev/feature/add_nick_huang

Add Nick Huang to contributors list

sudz123 committed:
Merge branch 'master' into patch-1

sudz123 committed:
Merge pull request #555 from Stell1c/patch-1

Added new details

sudz123 committed:
Merge pull request #558 from rohangaonk/patch-1

Update README.md

sudz123 committed:
Merge pull request #561 from Kaustubhprabhu98/patch-1

Basic EMI calculator

sudz123 committed:
Merge pull request #562 from mshoaib85/add-shoaib

add me to contributors list

sudz123 committed:
Merge branch 'master' into haappy-hacking

sudz123 committed:
Update Contributors.md

sudz123 committed:
Merge pull request #563 from vatsalvv/haappy-hacking

makin-a-new-world

kaffikaffi committed:
Removed excess exclamation marks

Removed two exclamation marks in the first sentence.

kaffikaffi committed:
Added name to Contributors.md

Added my Github-user

rakeshub4 committed:
update additional words

yarkalyba committed:
Add Yarka Lyba to Contributors.md

guido97 committed:
fix:minor typos

rodolfoip committed:
Add link to home page

DuskyPrawn committed:
Update Contributors.md

Happy Hactoberfest

DuskyPrawn committed:
Update Halloween.html

Removed repetitive sentences in Halloween's Origins section; capitalized Brazilians; added a missing space between sentences.

guido97 committed:
Create doc in Spanish

A Spanish version created for me (isn't translation) of a basic documentation for php

HectorHernandez19 committed:
Agregue mi nombre

Shubhajit01 committed:
Create HelloWorld.html

Hello world in Html

sudz123 committed:
Merge pull request #568 from kaffikaffi/master

Removed excess exclamation marks

sudz123 committed:
Merge pull request #570 from rakeshub4/patch-1

update additional words

sudz123 committed:
Merge branch 'master' into add-yarkalyba

sudz123 committed:
Merge pull request #571 from yarkalyba/add-yarkalyba

Add Yarka Lyba to Contributors.md

sudz123 committed:
Merge pull request #572 from guido97/patch-1

fix:minor typos

sudz123 committed:
Merge pull request #573 from rodolfoip/master

Add link to home page

sudz123 committed:
Merge branch 'master' into patch-1

sudz123 committed:
Merge pull request #574 from DuskyPrawn/patch-1

Update Contributors.md

sudz123 committed:
Merge pull request #575 from DuskyPrawn/patch-2

Update Halloween.html

sudz123 committed:
Merge pull request #576 from guido97/patch-2

Create doc in Spanish

sudz123 committed:
Merge pull request #577 from Enigma170/master

Agregue mi nombre

sudz123 committed:
Merge pull request #578 from Shubhajit01/patch-1

Create HelloWorld.html

ashishyadav400 committed:
Update Contributors.md

vamisola committed:
added myself to the contributors list

kasunSameera committed:
Update Contributors.md

diogoalexsmachado committed:
Fixe a typo :)

diogoalexsmachado committed:
Update Contributors.md

ayushg162 committed:
Adding my name to Contributors.md

NeoAnoman committed:
Adding my name to contibutors

Just to add my name to contributors.

Commit:
Merge pull request #605 from NeoAnoman/contro

Adding my name to contibutors

Commit:
Merge pull request #603 from ayushg162/ayush-goyal

Adding my name to Contributors.md

Commit:
Merge branch 'master' into patch-2

Commit:
Merge pull request #602 from diogoalexsmachado/patch-2

Update Contributors.md

Commit:
Merge pull request #601 from diogoalexsmachado/patch-1

Fixe a typo :)

Commit:
Merge branch 'master' into patch-1

Commit:
Merge pull request #596 from kasunSameera/patch-1

Update Contributors.md

Commit:
Merge branch 'master' into vamisola

Commit:
Merge pull request #595 from vamisola/vamisola

added myself to the contributors list

Commit:
Merge branch 'master' into patch-1

Commit:
Merge pull request #592 from ashishyadav400/patch-1

Update Contributors.md

Commit:
reduce some Css clutter

phyushin committed:
adding myself

jamescalixto committed:
added Mozilla developer docs as an additional resource

plaistos committed:
added Simon Plaistowe

Commit:
Merge pull request #616 from plaistos/bugger

added Simon Plaistowe

Commit:
Merge pull request #615 from jamescalixto/update

added Mozilla developer docs as an additional resource

Commit:
Merge pull request #614 from pixilala/tshirt

adding myself

Commit:
Merge pull request #612 from beattie282/reduceCssClutter

reduce some Css clutter

quangpro1610 committed:
create pull by quangpro1610

diptag committed:
Merge pull request #617 from quangpro1610/master

create pull by quangpro1610
